htsget-search/src/htsget/bam_search.rs

```diff
@@ -276,11 +277,11 @@ where
 
 #[cfg(test)]
 pub mod tests {
-
-  use super::*;
   use crate::htsget::Headers;
   use crate::storage::local::LocalStorage;
 
+  use super::*;
```

Comment:
Why this move?

Reply:
This seems to have been an auto import optimize by my IDE, however it does match the style of the other tests.
I can change it back if you would like?

Reply:
No, it's alright, I was just curious on whether it had to do with something else :)